project-base/storefront/components/Blocks/PromoCode/PromoCode.tsx

```diff
@@ -46,7 +46,7 @@ export const PromoCode: FC = () => {
             ) : (
                 <>
                     <Button
-                        className="text-sm mb-3"
+                        className="max-sm:w-full text-sm mb-3"
                         tid={TIDs.blocks_promocode_add_button}
                         variant="inverted"
```

Comment:
Based on this issue https://shopsys.atlassian.net/browse/SSP-2527 I would keep the original design with the primary variant. Clicking area is more obvious then.